### DISCUSSION TITLE
How to get a tuple of values ​​from the root?

### DISCUSSION THREAD

**Discussion by tnsr1** [sympy/sympy#26670] [Q&A]
Hello.
How to get a tuple of complex or real values ​​from the root?
For example of -4.
Thank you.

**Comment by sylee957** [comment]
`sqrt` does not return multiple values of root, because the mathematical definition of the function is to return the principal value of the root. 
If you need multiple values of root, you need to use `solve` or `solveset`:
`solveset(Eq(x**3, -4), x)`
